### PR TITLE
PSQLADM-428 Align Chapter headings

### PR DESCRIPTION
**Pull request by patrickbirch**
M  docs/configuring.md
M  docs/index.md
M  docs/install-v2.md
M  docs/installing-tarball.md
M  docs/installing-v1.md
M  docs/proxysql-changes.md
M  docs/proxysql-v1.md
M  docs/psa-build.md
M  docs/psa-config.md
M  docs/psa-options.md

### CODE REVIEW

File: docs/psa-config.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+# Example of the Percona Scheduler Admin configuration file
 
 *ProxySQL* with the `percona-scheduler-admin` stores the parameters in a
 configuration file that uses the toml format. This file defines
 the server credentials and other settings.
 
 Use the `--config-file` option to run the *percona-scheduler-admin* script.
 
-**NOTE**: It is not recommended to store the scheduler config file in the Home directory.
+!!! caution
```
Comment (alina-derkach-oaza) [inline docs/psa-config.md:9]: Maybe we should say !!! warning here? To have the same style everywhere.

Comment (patrickbirch) [reply docs/psa-config.md:9]: Fixed.

File: docs/install-proxysql2-tarball.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Install ProxySQL 2.x.x binary tarball
+
+if installing [*ProxySQL 2* with a package manager](install-v2.md) is not an option in your environment, you can install from a binary tarball. The difference between versions is the available tar files or the command to extract the file.
```
Comment (alina-derkach-oaza) [inline docs/install-proxysql2-tarball.md:3]: if installing - If installing

Comment (patrickbirch) [reply docs/install-proxysql2-tarball.md]: Fixed.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**